site/assets/_scss/site/objects/_components.scss

```diff
@@ -0,0 +1,81 @@
+@mixin breakpoint($point) {
```

Comment:
@amymanion like the other updates you made - use the existing bootstrap mixin; or if you need this put it in a `_mixins.scss` partial

Reply:
@ccbayer Good catch - I was trying to catch all of the previous updates, but missed this one.